### ISSUE TITLE
Spanish special chars are overridden with some other ASCII values

### ISSUE THREAD

**Issue by raydeng83** [ForgeRock/fr-config-manager#203]
When I was pushing ESVs with Spanish special chars like é, á, í, ó, and ú, they are overridden with some random ASCII values. Thinking we may need to add support for Special Chars.

**Comment by christian-brindley** [comment]
Hi @raydeng83 - please can you provide some more details about how you are defining the values? I can't reproduce the issue - seems to work okay in the test environment, so wondering if this is an encoding problem in your environment? Would appreciate details like

- Is this a variable or secret
- How is the value stored - in a `.env` file, or pipeline environment variable etc
- Is it hardcoded in the ESV json file, or as an env variable
- Anything else relevant

If it is an env variable, you could try base64 encoding and specify using the base64 prefix - e.g.

- If value is "éáíóú"
- Env variable set to `ESV_INTERNATIONAL=w6nDocOtw7PDug==`
- Variable JSON file as follows

```
{
  "_id": "esv-international",
  "description": "Test international chars",
  "expressionType": "string",
  "valueBase64": "${BASE64:ESV_INTERNATIONAL}"
}
```

**Comment by kashif9to5** [comment]
Hi Christian, i am having the same issue. this issue is only applicable when using fr-config pull, that Spanish characters are not preserved. How to ensure that. Converting it to base64 would not be easy maintenance.

**Comment by christian-brindley** [comment]
Hi @kashif9to5 

The fr-config-pull command does not export the value of the ESV, only the ESV configuration. Unless you mean the output it gives when you run the tool with the `--dump` option? Do you have an end to end example of the ESV you create, the fr-config-pull command that you use to export, and the place where the characters are corrupted?

Thanks

Christian

**Comment by kashif9to5** [comment]
That's right, using --dump it doesn't retain Spanish characters. Some example below

ESV_ERRORINVALIDOTPEMAILSMSVOICE_ES='CÃ³digo no vÃ¡lido. Por favor intÃ©ntalo de nuevo.'

**Comment by christian-brindley** [comment]
Ah - understood. I will fix the pull script to handle non-English characters properly when dumping variable values. I'll raise a separate issue to make sure we handle characters properly elsewhere when they are base64 encoded.

**Comment by christian-brindley** [comment]
Created pull request

   https://github.com/ForgeRock/fr-config-manager/pull/new/203-esv-pull-international-chars
